modules/installation/pages/proxy-setup.adoc

```diff
@@ -207,45 +207,36 @@ filename: "pxelinux.0"
 
 
 
-// REMARK: 2019-08-23, ke: this needs closer checking
-// see https://github.com/SUSE/spacewalk/issues/10521
-// and https://github.com/uyuni-project/uyuni-docs/issues/110
-[[replacing.a.susemgrproxy]]
+[[replace-susemgrproxy]]
 == Replace a {susemgrproxy}
 
-A {susemgrproxy} is dumb in that it does not contain any information about the clients that are connected to it.
-A {susemgrproxy} can therefore be replaced by a new one.
-Naturally, the replacement proxy must have the same name and IP address as its predecessor.
+A proxy does not contain any information about the clients that are connected to it.
+Therefore, a proxy can be replaced by a new one at any time.
+The replacement proxy must have the same name and IP address as its predecessor.
 
-In order to replace a {susemgrproxy} and keeping the clients registered to the proxy leave the old proxy in {productname}.
+Leave the old proxy in place while you prepare the replacement.
 Create a reactivation key for this system and then register the new proxy using the reactivation key.
 If you do not use the reactivation key, you will need to re-register all the clients against the new proxy.
 
 [[proxy.migration3.replace]]
-.Procedure: Replacing a {susemgrproxy} and Keeping the Clients Registered
+.Procedure: Replacing a Proxy and Keeping the Clients Registered
 . Before starting the actual migration procedure, save the data from the old proxy, if needed.
-Consider copying important data to a central place that can also be accessed by the new proxy.
-// REMARK: 2019-08-23, ke: this is unclear
-// ** Copy the scripts that are still needed.
-// ** Copy the activation keys from the previous server.
-// Of course, it is always better to re-create the keys.
+Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
 . Shut down the proxy.
 . Install a new {susemgrproxy}{nbsp}{productnumber}, following xref:install-proxy-unified.adoc[Proxy Installation].
```

Comment:
Shouldn't we use here:
```
. Install a new proxy{nbsp}{productnumber}, following xref:install-proxy-uyuni.adoc[Uyuni Proxy Installation].
```
Just as you did for Uyuni?

Reply:
I do not think so.  I think the preferred way to install the SUMA Proxy is to use the Unified Installer.